src/main/java/com/cloudbees/dockerpublish/DockerBuilder.java

```diff
@@ -254,7 +274,25 @@ private Result executeCmd(String cmd) throws IOException, InterruptedException {
                     .stderr(stderr)
                     .cmdAsSingleString(cmd)
                     .start().join() == 0;
-            String stdoutStr = stdout.toString();
+
+            // capture the stdout so it can be parsed later on
+            String stdoutStr = null;
+            try {
+                Computer computer = Computer.currentComputer();
+                if (computer != null) {
+                    Charset charset = computer.getDefaultCharset();
+                    if (charset != null) {
```

Comment:
that was addressing @rsandell comment. Based in other feedback it may be pointless anyway when running in a slave, but I'm not sure